src/views/login/index.js

```diff
@@ -45,8 +65,9 @@ const Login = () => {
   useEffect(() => {
     if (accessToken) {
       dispatch(getUser())
+      navigate('/dashboard')
```

Comment:
There's no need to get the user here. You should get it on the app or dashboard views